navigation/page-index/page-index.js

```diff
@@ -0,0 +1,27 @@
+module.exports = [
+  {
+    title: "Find a docs page",
+    href: "navigation",
+    filePath: "index.md",
+    children: [
+      {
+        title: "Find pages by Cloud Console location",
+        overrideHref: "/cloud-console",
+        excerpt: "Browse topics by Timescale Cloud console location.",
+        type: "placeholder",
+      },
+      {
+        title: "Find by keyword",
+        overrideHref: "/keywords",
+        excerpt: "Browse topics by keywords.",
+        type: "placeholder",
+      },
+      {
+        title: "Full search",
+        overrideHref: "/search",
```

Comment:
When you click this in preview, it gives you a no results page. Can we preload it with a search term? "Timescale" maybe?

Reply:
yeah I'm going to redo the empty state 👍🏼

Reply:
Excellent 😁